action.yml

```diff
@@ -64,8 +64,8 @@ inputs:
     description: "verify the generated signatures after signing"
     required: false
     default: true
-  verify-cert-email:
-    description: "verify the email address to in the signing certificate's Subject Alternative Name (only applies when `verify` is on)"
+  verify-cert-identity:
+    description: "verify the identity in the signing certificate's Subject Alternative Name (only applies when `verify` is on)"
```

Comment:
I _think_ we want to do a smooth deprecation here: we should treat `verify-cert-email` as an alias for `verify-cert-identity`, and emit a warning when the user uses it instead of the latter.